ENH make the status live!

beckermr opened the pull request:
This PR closes #65. 

I could get 

 - [x] circle
 - [x] travis
 - [x] appveyor
 - [x] staged-recipes
 - [x] github
 - [x] docs

I could not get

- [ ] anaconda.org
- [ ] the linter bot
- [ ] azure
- [ ] drone

beckermr commented:
here are some screen shots:

![Uploading Screen Shot 2020-01-30 at 11.52.42 PM.png…]()

<img width="853" alt="Screen Shot 2020-01-30 at 11 52 53 PM" src="https://user-images.githubusercontent.com/5296416/73515649-b00feb00-43bb-11ea-86a5-84d0000ec4b5.png">

beckermr commented:
@conda-forge/core @conda-forge/bot This one is ready to go I think. I am happy to pass on the snarky title, but it seemed like fun. ;P

CJ-Wright commented:
Personally I would put the cloud services lower, those are usually a given (hopefully!)

beckermr commented:
Will do!

isuruf commented:
I agree. I think the repos and bots section should be first, then cloud, then migrations

beckermr commented:
Alright @CJ-Wright @isuruf the current order is

1. repos and bots
2. CDN
3. cloud
4. migrations
5. usage

Does this look good?

beckermr commented:
<img width="864" alt="Screen Shot 2020-01-31 at 8 27 20 AM" src="https://user-images.githubusercontent.com/5296416/73546844-85e21b80-4403-11ea-9a14-9d3320784dca.png">

CJ-Wright commented:
Personally I mostly use the status page to look at the current migration status, but I might not be the ideal user to design for.

beckermr commented:
I'd say "design" is a pretty hefty word for what I am doing here. ;p

In any case, I don't really care either way, so whatever you and @isuruf want to do is fine.

CJ-Wright commented:
@conda-forge/core thoughts?

isuruf commented:
Usually `migrations` needs some scrolling because there are many. Even if the statuses are moved to the top, migrations will still be there in the screen right?

beckermr commented:
Not quite @isuruf here is what I have

![status](https://user-images.githubusercontent.com/5296416/73548169-ec683900-4405-11ea-9243-9b30075c05c7.gif)

CJ-Wright commented on template.html:
```diff
@@ -38,7 +38,7 @@ <h1 class="title">{{ config.title }}</h1>
         <!-- {% if not panels %}
         <div class="panel operational" data-l10n-id="systems-operational">
-          All Systems Operational
+          All Systems Mostly Operational
```
I feel like this is akin to "mostly harmless"

beckermr replied:
Happy to remove! It is just weird because we don't update that.

CJ-Wright replied:
I didn't mean remove, I just thought the parallels with hitchhikers guide were funny

beckermr replied:
ahhhh putting it back!

beckermr commented:
OK @isuruf @CJ-Wright I reworked the page to a compromise that gets the migrations near the top. I moved the CDN status to the "repos and bots" section since it made more sense there and saved space. 

![status](https://user-images.githubusercontent.com/5296416/73553869-4a9a1980-4410-11ea-8797-07edf69b32eb.gif)

beckermr commented:
Any other comments here @conda-forge/core?
